python-package/lightgbm/basic.py

```diff
@@ -1245,12 +1245,12 @@ def __getstate__(self):
         return this
 
     def __setstate__(self, state):
-        model_str = state.get('handle', None)
+        model_str = state.get('handle', None)  # list of str
```

Comment:
![image](https://cloud.githubusercontent.com/assets/433434/22285718/289952ea-e330-11e6-8a3b-644fde9652aa.png)
Please check this.
_I'm not sure_